tools-v2/pkg/cli/command/curvebs/update/peer/peer.go

```diff
@@ -104,14 +104,18 @@ func (pCmd *PeerCommand) AddFlags() {
 func ParsePeer(peer string) (*common.Peer, *cmderror.CmdError) {
 	cs := strings.Split(peer, ":")
 	if len(cs) != 3 {
-		pErr := cmderror.ErrGetPeer()
-		pErr.Format(fmt.Sprintf("error format for the peer info %s", peer))
+		pErr := func() *cmderror.CmdError {
+			return cmderror.NewInternalCmdError(15, "split peer [%s] failed, peer should be like: 127.0.0.1:8200:0")
```

Comment:
This is to modify the file [error.go](https://github.com/opencurve/curve/blob/master/tools-v2/internal/error/error.go)
```go
	ErrSplitPeer = func() *CmdError {
		return NewInternalCmdError(15, "split peer[%s] failed, peer should be like: 127.0.0.1:6700:0")
	}
```